kevm_pyk/src/kevm_pyk/solc_to_k.py

```diff
@@ -245,7 +245,13 @@ def gen_claims_for_contract(empty_config: KInner, contract_name: str, calldata_c
         'PC_CELL': intToken(0),
         'GAS_CELL': KEVM.inf_gas(KVariable('VGAS')),
         'K_CELL': KSequence([KEVM.execute(), KVariable('CONTINUATION')]),
-        'ACCOUNTS_CELL': KApply('_AccountCellMap_', [account_cell, KVariable('ACCOUNTS')]),
+        # KApply('_AccountCellMap_', [account_cell, KVariable('ACCOUNTS')]),
+        # KEVM.accounts([account_cell, KEVM.foundry_account()])
+        'ACCOUNTS_CELL': KEVM.accounts([account_cell,
+                                        KEVM.account_CALLER(),
+                                        KEVM.account_TEST_CONTRACT_ADDRESS(),
+                                        KEVM.account_CHEATCODE_ADDRESS(),
+                                        KEVM.account_HARDHAT_CONSOLE_ADDRESS()])
```

Comment:
```suggestion
        'ACCOUNTS_CELL': KEVM.accounts([
            account_cell,
            KEVM.account_CALLER(),
            KEVM.account_TEST_CONTRACT_ADDRESS(),
            KEVM.account_CHEATCODE_ADDRESS(),
            KEVM.account_HARDHAT_CONSOLE_ADDRESS()
        ]),
```

Reply:
We keep the trailing commas on purpose, and have everything on newlines inbetween the start and end `[ ... ]` on purpose.
This makes it much easier to just copy/paste and re-sort the elements.